test/kubemark/start-kubemark.sh

```diff
@@ -113,13 +113,30 @@ function create-kube-hollow-node-resources {
   # Create kubemark namespace.
   "${KUBECTL}" create -f "${RESOURCE_DIRECTORY}/kubemark-ns.json"
 
+  # Create kubemark configmap and secret
```

Comment:
no need to have this extra code, the different RP masters are with differnt kubeconfig files, and the hollow-node function can be called multiple times with those different kubeconfig files to parepare the hollow-nodes